lib/LLVMIRCodeGen/DebugInfo.cpp

```diff
@@ -160,18 +160,10 @@ LLVMIRGen::getOrCreateFunctionDebugInfo(llvm::Function *F, llvm::DIScope *scope,
     auto *DIFunctionTy = DIBuilder_->createSubroutineType(
         DIBuilder_->getOrCreateTypeArray(paramTys));
     // Create a debug information for the current function.
-#if LLVM_VERSION_MAJOR < 8 || FACEBOOK_INTERNAL
     DIFunction = DIBuilder_->createFunction(
         scope, F->getName(), "", file, lineNo, DIFunctionTy,
         false /* internal linkage */, true /* definition */, lineNo,
         llvm::DINode::FlagPrototyped, true /* isOptimized */);
-#else
```

Comment:
Why did you remove this snippet? This was introduced for LLVM >=8, IIRC.

Reply:
This passed in the internal version, therefore I wrongfully assumed it passes for llvm8. It doesn't built locally. reverting all of these patches.